### ISSUE TITLE
The error of example about LIFCell ?

### ISSUE THREAD

**Issue by luciddream-tsin** [norse/norse#279]
Hello, I need the function that give x dim input and y dim output,
like the example in the document following, but when I run this , 
I get a error, obviously the document is old, so, how I do this at 
new version 0.07?
```
data = torch.zeros(5, 2) # 5 batches, 2 neurons
l = LIFCell(2, 4)
l(data) # Returns tuple of (Tensor(5, 4), LIFState)
```

**Comment by Pugavkomm** [comment]
Try this:
`
data = torch.zeros(5, 2) # 5 batches, 2 neurons
l = LIFCell()
l(data) # Returns tuple of (Tensor(5, 4), LIFState)
`

**Comment by luciddream-tsin** [comment]
Oh, thanks very much for reply!
Could you tell me more detail about the connection of LIFRecurrent?
It is a fully connection?

**Comment by Pugavkomm** [comment]
It is detailed [here](https://norse.github.io/norse/generated/norse.torch.LIFRecurrent.html). This layer has input weights and recurrent weights of each neuron in this layer. If you need to disable connections between neurons inside the layer, you can set a zero tensor (`torch.zeros`).

**Comment by luciddream-tsin** [comment]
ok, thanks again!
